pandas/core/reshape.py

```diff
@@ -1213,7 +1216,7 @@ def make_axis_dummies(frame, axis='minor', transform=None):
         labels = cat.codes
         items = cat.categories
 
-    values = np.eye(len(items), dtype=float)
+    values = np.eye(len(items), dtype=np.uint8)
```

Comment:
Just doing `uint` went to a uint64 on my machine. I'm just going to revert this change. As far as I can tell this only affects the user publicly via `pd.ols`. Given that we're deprecating that, it doesn't make sense to change it now.